docs/usersguide/migrating.rst

```diff
@@ -14,12 +14,12 @@ currently installed libraries:
 
 .. code-block:: bash
```

Comment:
If you use `$ ...` then the `code-block` should be `console`, see http://build-me-the-docs-please.readthedocs.io/en/latest/Using_Sphinx/ShowingCodeExamplesInSphinx.html#pygments-lexers